dockstore-client/src/main/java/io/dockstore/client/cli/nested/WESLauncher.java

```diff
@@ -13,18 +14,19 @@
 import io.openapi.wes.client.api.WorkflowExecutionServiceApi;
 import io.openapi.wes.client.model.RunId;
 import org.apache.commons.io.FileUtils;
-import org.apache.commons.io.FilenameUtils;
 import org.apache.commons.lang3.tuple.ImmutablePair;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import static io.dockstore.client.cli.ArgumentUtility.exceptionMessage;
+import static io.dockstore.client.cli.Client.GENERIC_ERROR;
 import static io.dockstore.client.cli.Client.IO_ERROR;
 
 public class WESLauncher extends BaseLauncher {
     private static final Logger LOG = LoggerFactory.getLogger(WESLauncher.class);
     private static final String TAGS = "WorkflowExecutionService";
     private static final String WORKFLOW_TYPE_VERSION = "1.0";
+    private static final String WDL_WORKFLOW_TYPE_VERSION = "draft-2";
```

Comment:
This is the version that Dockstore (sort of) supports. However, we do allow versions greater than draft-2, even if we don't support all the features.

Does a WDL file have a version in it, like CWL? I didn't see it off-hand, but if it does, it seems like the version should come from the particular workflow in question, and not be a global value.

Reply:
Is worth (code) documenting what this value is for anyway